seqr/views/utils/test_utils.py

```diff
@@ -222,7 +224,7 @@ def _check_login(self, url, permission_level, request_data=None):
         'genotypes': {
             'NA19675': {
                 'sampleId': 'NA19675',
-                'ab': 0.7021276595744681,
+                'ab': 0.702127659574,
```

Comment:
Removed 4 trailing digits in the decimal because Python 3 supports fewer decimals.